txpool/txpool.go

```diff
@@ -663,13 +672,23 @@ func (p *TxPool) validateTx(tx *types.Transaction) error {
 		if tx.GasFeeCap().Cmp(new(big.Int).SetUint64(baseFee)) < 0 {
 			metrics.IncrCounter([]string{txPoolMetrics, "underpriced_tx"}, 1)
 
+			p.logger.Debug("dynamic tx is undepriced",
+				"gasPrice", tx.GetGasPrice(baseFee).String(),
```

Comment:
Did you want to log the gas fee cap here?

```suggestion
				"gasFeeCap", tx.GasFeeCap().String(),
```

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/194/commits/b51efd8cd6fa2396c81c0d58a3fd7c578639f8d4